packages/flutter_driver/lib/src/enum_util.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Indexes a list of `enum` values by simple name.
+///
+/// In Dart enum names are prefixed with enum class name. For example, for
+/// `enum Vote { yea, nay }`, `Vote.yea.toString()` produces `"Vote.yea"`
+/// rather than just `"yea"` - the simple name.
+///
```

Comment:
this is missing another sentence that starts "This class..." and says what the class does instead. :-)